src/app_engine/gcmrecord.py

```diff
@@ -97,7 +102,9 @@ def parse_json(cls, json_string):
       content_dictionary = json.loads(json_string)
       required_keys = [
           cls.USER_ID_KEY, cls.GCM_ID_KEY, cls.CODE_KEY, cls.CODE_SENT_TIME_KEY,
-          cls.VERIFIED_KEY, cls.LAST_MODIFIED_TIME_KEY
+          cls.VERIFIED_KEY, cls.LAST_MODIFIED_TIME_KEY,
+          # TODO(tkchin): add registration id once we clear the store.
+          # cls.REGISTRATION_ID_KEY,
```

Comment:
Should we clear the store on pushing this change? Old registrations without the new field will fail to receive calls once the client updates.